lib/bintray_client.rb

```diff
@@ -0,0 +1,53 @@
+require 'net/http'
+require 'json'
+
+class BintrayClient
+
+  def initialize(user, key, base_url = 'https://bintray.com/api/v1/packages')
+    @base_url = base_url
+    @user = user
+    @key = key
+  end
+
+  def get_bintray_versions(project)
+    uri = URI("#{@base_url}/#{project}")
+    http = Net::HTTP.new(uri.host, uri.port)
+    http.use_ssl = true
+    request = Net::HTTP::Get.new(uri.request_uri)
+
+    response = http.request(request)
+
+    return nil unless response.kind_of? Net::HTTPSuccess
```

Comment:
Do you think it could be worth extracting those lines in a small helper method, given that the same is used for the `GitHubClient` (and a very similar one is used for `delete_bintray_version` except for the `Net::HTTP::Delete`)?

Reply:
Love a good DRY refactor suggestion. Thanks!